.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,38 @@
+on:
+  - workflow_dispatch
+  - push
+
+jobs:
+  e2e-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "18.16.0"
+
+      - name: Install
+        run: yarn install
+
+      - name: Build
+        run: yarn build
+
+      - name: Test
+        env:
+          LOG_ENVIRONMENT: "production"
+          SUPABASE_URL: ${{ secrets.SUPABASE_URL }}
+          SUPABASE_KEY: ${{ secrets.SUPABASE_KEY }}
+          X25519_PRIVATE_KEY: "QCDb30UHUkwJAGhLWC-R2N0PiEbd4vQY6qH2Wloybyo"
```

Comment:
Should this be exposed? Not sure exactly what it is used for.

Reply:
it's okay, it must be the same because the test's config wallet private key in encrypted with this private key